### ISSUE TITLE
No compress in SVG format

### ISSUE THREAD

**Issue by Erimus-Koo** [jgraph/drawio#4204]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the solution you'd like**
I use draw.io in my markdown notes and save the diagrams as SVG files to keep them editable and insert them into markdown as images, which works perfectly. 

However, the SVG format does not have a "compress" option like the "drawio" format does. This makes it less friendly for GIT, which I use for my markdown project.

Could you please consider adding a compress option to keep the XML in separate lines? This would make it more GIT-friendly, and I would greatly appreciate it.

**Comment by davidjgraph** [comment]
I'm not clear what you're asking. Currently, the embedded XML is not compressed, you want an option to compress it?

**Comment by Erimus-Koo** [comment]
## .draw.io
While I create a new file with extension "drawio", it's not compressed. It's GIT-friendly.
<img width="317" alt="image" src="https://github.com/jgraph/drawio/assets/17139670/1dec7a4b-daa8-4d24-b747-b747d693d6f0">
<img width="763" alt="image" src="https://github.com/jgraph/drawio/assets/17139670/5dd55946-e0e6-49f4-972d-6d32f15c4d8d">


## .svg
If I save it as a SVG file.
<img width="302" alt="image" src="https://github.com/jgraph/drawio/assets/17139670/9c544df6-efac-45b7-909d-fff71791663b">
<img width="289" alt="image" src="https://github.com/jgraph/drawio/assets/17139670/a836196a-7cbe-45b0-be88-ab6faecc03a2">

First, the menu item "properties" is gone.
<img width="292" alt="image" src="https://github.com/jgraph/drawio/assets/17139670/6380f281-dcb9-4141-bd7d-b3ca24d2a795">

And the SVG content is all in one line, that's not GIT-friendly.
<img width="753" alt="image" src="https://github.com/jgraph/drawio/assets/17139670/76877283-5857-4f60-a292-ca80dff34570">

The SVG file can be easily inserted as an image into the markdown file, which is great.

In summary, I am looking for a solution to easily insert the file into the markdown and make it GIT-friendly. Do you have any ideas or advice?

**Comment by davidjgraph** [comment]
OK, I understand, you want an option to not minimise the content, rather than not compress it.
